doc-source/notifiers.rst

```diff
@@ -5,7 +5,7 @@
 Notifiers
 =========
 
-The last phase in a typical compliance-tool check run is the notification
+The last phase in a typical framework check run is the notification
 system.  Multiple notifiers can be targeted as part of this phase  by using
 the ``--notify`` option on the ``compliance --check`` command.  Valid
 notifier options are ``stdout``, ``slack``, ``pagerduty``, ``ghe_issues`` and,
```

Comment:
Change `ghe_issues` to `gh_issues` and also add `findings`.  I'm wondering if removing the options altogether would be a good idea?  This way when new notifiers are added we don't need to remember to update the docs.